### PR TITLE
raidboss: Add some basic Tower of Zot triggers

### PR DESCRIPTION
**Pull request by valarnin**
Haven't tested, WIP, etc etc. Just putting it out there to make sure directory structure lines up with what's expected.

### CODE REVIEW

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: 0x3B8, // @TODO: change to zoneid entry when available
+  triggers: [
+    // Tankbuster shared by 2nd and 3rd bosses
+    {
+      id: 'Isitva Siddhi',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '62A9', source: '(?:Sanduruva|Minduruva)', capture: true }),
```
Comment (JLGarber) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts:16]:
```suggestion
      netRegex: NetRegexes.startsUsing({ id: '62A9', source: ['Sanduruva, 'Minduruva'], capture: true }),
```

Not that it matters *that* much, but it's probably better not to use raw regex syntax within regex helpers.

**Comment by valarnin** [comment]
Sorry about the force push, was still sorting out some issues with the new windows install and that caused all sorts of weird issues on my end with the git tree.

This is technically functional now, validated in-zone and via raidemulator. I'll mark it as Ready for review, feel free to merge or to wait until there's other triggers available.

**Comment by JLGarber** [comment]
Oh, sorry, I should have mentioned it previously. Tank busters no longer take conditions, they're now intended for all users. (Individuals can turn them off as they like.)

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: 0x3B8, // @TODO: change to zoneid entry when available
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts:10]: This should already exist!

Comment (valarnin) [reply ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts]: My local tree was pre-EW-update when I wrote these originally.

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '62C0', source: 'Sanduruva', capture: true }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    // Trash mob tankbuster
+    {
+      id: 'ToZ Wheel',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '5E52', source: 'Zot Roader', capture: true }),
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts:28]: It looks like there's no startsUsing for this attack, it's just an uncasted ability.

Comment (valarnin) [reply ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts:28]: Hmm, I guess it'll need to be dropped from triggers then. I wanted to include it due to how hard it hits unmitigated but with no startsCasting I'm not sure if there's a feasible way to warn for it.

Comment (JLGarber) [reply ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts:28]: I'll try to generate timelines for this tonight and we'll see whether we can do a timeline trigger instead.

Comment (JLGarber) [reply ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts:28]: That isn’t a boss, and I am sad now. T_T

**Pull request merged**